deploy/test/test_update_alerts_websocket_v3.py

```
@@ -0,0 +1,482 @@
+    )
+    mock_get_regions.return_value = (districts)
+    mock_get_cache_data.return_value = [[2, 1700000000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], 
+                        [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], 
+                        [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], 
+                        [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], 
+                        [0, 1645674000], [0, 1645674000]
+                        ]
+
+    await update_alerts_websocket_v3(mock_mc, run_once=True)
+
+    expected_result = [[2, 1700000000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], 
```

Ruff (0.8.2)

Local variable `expected_result` is assigned to but never used

Remove assignment to unused variable `expected_result`

(F841)